ext/MeasurementsJunoExt.jl

```diff
@@ -0,0 +1,28 @@
+module MeasurementsJunoExt
+
+@static if isdefined(Base, :get_extension)
```

Comment:
Top-level `@static` does nothing, the expression is "statically" evaluated anyway
```suggestion
if isdefined(Base, :get_extension)
```